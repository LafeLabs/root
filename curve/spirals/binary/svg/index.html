<p><a href = "svg1534812851.svg"><img src = "svg1534812851.svg"></a></p>
<p><a href = "svg1531522562.svg"><img src = "svg1531522562.svg"></a></p>
