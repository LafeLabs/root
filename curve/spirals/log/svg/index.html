<p><a href = "svg1534812857.svg"><img src = "svg1534812857.svg"></a></p>
