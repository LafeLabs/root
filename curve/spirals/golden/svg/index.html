<p><a href = "svg1532032772.svg"><img src = "svg1532032772.svg"></a></p>
<p><a href = "svg1531180313.svg"><img src = "svg1531180313.svg"></a></p>
<p><a href = "svg1531179612.svg"><img src = "svg1531179612.svg"></a></p>
